.github/workflows/draft-pdf.yml

```
@@ -0,0 +1,21 @@
+on:
+  # Triggers the workflow on push event for paper branch
+  push:
+    branches: [ "paper" ]
+jobs:
+  paper:
+    runs-on: ubuntu-latest
+    name: Paper Draft
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

I'm not 100% sure but I think we need to specify the branch here.
```suggestion
        uses: actions/checkout@v4
        with:
          ref: paper
```